src/orm/orm.module.ts

```diff
@@ -13,7 +14,7 @@ import { User } from '../model/user.entity';
         new OrmConfigFactory(configService).buildConfig(),
     }),
     MikroOrmModule.forFeature({
-      entities: [Testimonial, User],
+      entities: [Testimonial, User, Crystal],
```

Comment:
Please rename corresponding classes too
```suggestion
      entities: [Testimonial, User, Product],
```